components/Navigation/index.js

```diff
@@ -42,7 +42,7 @@ const StyledIconBox = styled(Link)`
 export default function Navigation() {
   return (
     <>
-      <NavBar>
+      <NavBar $navbar="true">
```

Comment:
Just curious, is it considered conventional to add the dollar sign for styled component props? Seems cryptic to the eye.

Reply:
Hey @Kheirah .
I got a yellow message in the console saying `styled-components: it looks like an unknown prop "xxx" is being sent through to the DOM` and `consider using transient props (`$` prefix for automatic filtering.)`. So I've done as suggested and the warning disappeared.